registry/storage/ocimanifesthandler.go

```diff
@@ -109,7 +118,7 @@ func (ms *ocischemaManifestHandler) verifyManifest(ctx context.Context, mnfst oc
 				}
 			}
 
-		case v1.MediaTypeImageManifest:
+		case v1.MediaTypeImageManifest, v1.MediaTypeArtifactManifest, v1.MediaTypeImageIndex, schema2.MediaTypeManifest:
```

Comment:
how about docker manifest list?

Reply:
Added manifest list and schema1 manifest.